### PR TITLE
feat(tab-view-android): enable tabs repositioning

### PR DESCRIPTION
**Pull request by ADjenkov**
Changes:
1. Add `androidTabsPosition` property that enables positioning the tab view items `top` or `bottom`. 
2. Switch to the new [`TabViewPager`](https://github.com/NativeScript/tns-core-modules-widgets/pull/112) in order to disable swipe gestures over the content area (when  `androidTabsPosition="bottom"`.

### CODE REVIEW

**Comment by ADjenkov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by ADjenkov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by ADjenkov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by ADjenkov** [comment]
test uitests widgets#djenkov/tab-view-pager

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager testsappng

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager testsappng

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager testsappng cuteness functional-tests#tsenov/tabview-uitest

File: tns-core-modules/ui/tab-view/tab-view.android.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
     TabViewBase, TabViewItemBase, itemsProperty, selectedIndexProperty,
     tabTextColorProperty, tabBackgroundColorProperty, selectedTabTextColorProperty,
     androidSelectedTabHighlightColorProperty, androidOffscreenTabLimitProperty,
-    fontSizeProperty, fontInternalProperty, View, layout,
+    androidTabsPositionProperty, fontSizeProperty, fontInternalProperty, View, layout,
```
Comment (manoldonev) [inline tns-core-modules/ui/tab-view/tab-view.android.ts]: Shouldn't this be referenced somewhere if it is imported here?

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager testsappng cuteness functional-tests#tsenov/tabview-uitest

**Comment by SvetoslavTsenov** [comment]
test uitests widgets#djenkov/tab-view-pager testsappng cuteness functional-tests#tsenov/tabview-uitest

**Comment by SvetoslavTsenov** [comment]
test uitests testsappng cuteness functional-tests#tsenov/tabview-uitest

**Comment by SvetoslavTsenov** [comment]
Threre are two failing tests:
http://nsbuild01.telerik.com:8080/build/job/integration-tests-stable-cuteness-android-api23-emulator/1593/

**Comment by SvetoslavTsenov** [comment]
test cuteness

**Comment by SvetoslavTsenov** [comment]
👍

**Pull request merged**